### ISSUE TITLE
JAWS incorrectly announces toolbar label "Toolbar for code block which can be reached via tab" information pulled from ARIA attributes:A11y_Visual Studio Code Client_get started with accessibility feature_Usable

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#247098]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

## GitHub Tags:
#A11yeDAD; #A11yTCS; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #A11yUsable; #Keyboard; #FTP; #Win32; #JAWS;

## Environment and OS details:
Visual Studio Code version 1.99.2
Microsoft Windows 11 Enterprise (24H2)
OS Build 26100.2605
JAWS version 2025

## Repro Steps:

1. Turn on JAWS screen reader
2. Open Visual studio code insider
3. Link a GitHub Account
4. Sign Up for Free Copilot Use
5. Sign Up for Free Copilot Use and in copilot chat copilot response containing one or more code blocks.
6. TAB till tool bar of code block.
7. Observe that JAWS incorrectly announces toolbar label "Toolbar for code block which can be reached via tab" information pulled from ARIA attributes

## Actual:
JAWS reads and announces a label such as “Toolbar for code block which can be reached via tab”, which is being pulled from the element's ARIA-label or label attribute

## Expected:
JAWS should announce the toolbar with a clear and accurate label that aligns with the visual context and accessibility best practices. If the label is no longer valid or intended to be removed/changed, JAWS should not continue to pick it up from outdated or residual ARIA attributes after updates.

## User Impact:
if screen reach announces tool bar label as "Toolbar for code block which can be reached via tab" after reaching to tool bar than there is user of it for screen reader user.

**Comment by kapilvaishna** [comment]
Issue is not repro in below build so closing the bug.
Visual studio code insider
Version 1.100.0

https://github.com/user-attachments/assets/4a00be49-5e1a-4446-a3a5-d7396751db37
